packages/rpc-provider/src/ws/Provider.ts

```diff
@@ -75,6 +75,8 @@ export default class WsProvider implements WSProviderInterface {
 
   readonly #waitingForId: Record<string, JsonRpcResponse> = {};
 
+  #endpointNext: number;
```

Comment:
Please just move this down to below `autoConnectMs` (keeping it alphabetical in the mutable section)